watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/CreateTaskFragment.java

```diff
@@ -47,16 +48,21 @@ public CreateTaskFragment() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         mNetworkManager = NetworkManager.getInstance(getActivity().getApplicationContext());
-        mMainActivity = (MainActivity) getActivity();
     }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View view =  inflater.inflate(R.layout.fragment_create_task, container, false);
-
-        setButtonListeners(view);
-        return view;
+        super.onCreateView(inflater, container, savedInstanceState);
+        mParentActivity = (MainActivity) getActivity();
+        return inflater.inflate(R.layout.fragment_create_task, container, false);
+    }
+    
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        setButtonListeners();
+        setFocusListeners();
```

Comment:
If these methods are in onCreateView, they will also need to take in a View as an argument. getView() won't work at that stage.